### DISCUSSION TITLE
Alpine Floating Tags Updated to Alpine 3.15

### DISCUSSION THREAD

**Discussion by mthalman** [dotnet/dotnet-docker#3705] [Announcements]
# Alpine Floating Tags Updated to Alpine 3.15

In the March servicing release, [Alpine 3.15 container images were published](https://github.com/dotnet/dotnet-docker/issues/3550). For the April servicing release, we've updated all Alpine floating tags from Alpine 3.14 to Alpine 3.15. This update follows our [tagging policy](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-tags.md).

## Details

The affected floating tags use this naming pattern:

* `<version>-alpine` (e.g. `6.0-alpine`)
* `<version>-alpine-<arch>` (e.g. `6.0-alpine-amd64`)

Related to https://github.com/dotnet/dotnet-docker/issues/3551, commit: 75ab725

The following Docker repos have been updated:

* [dotnet/sdk](https://hub.docker.com/_/microsoft-dotnet-sdk)
* [dotnet/aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet)
* [dotnet/runtime](https://hub.docker.com/_/microsoft-dotnet-runtime)
* [dotnet/runtime-deps](https://hub.docker.com/_/microsoft-dotnet-runtime-deps)

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.
